src/modules/dashboard/components/UserPermissions/UserPermissions.tsx

```diff
@@ -34,7 +36,7 @@ const UserPermissions = ({ roles, directRoles, appearance }: Props) => {
   const [headRole, ...restRoles] = sortedRoles;
   return (
     <div className={getMainClasses(appearance, styles)}>
-      {headRole && (
+      {!isNil(headRole) && (
```

Comment:
I don't really like us relying on `lodash`, so please try to avoid it as much as possible

I'm fine with this for now and you can leave it in, just try to avoid it in the future.